R/pruneRepo.R

```diff
@@ -98,30 +129,55 @@ pruneRepo <- function(repopath = getOption("dratRepo", "~/git/drat"),
     ## 8   RcppTOML_0.0.3.tar.gz  RcppTOML    0.0.3   TRUE
     ## 9  winsorize_0.0.2.tar.gz winsorize    0.0.2   TRUE
     ## R>
+    df
+}
 
-    haspkg <- requireNamespace("git2r", quietly = TRUE)
-    
+##' @rdname pruneRepo
+pruneRepo <- function(repopath = getOption("dratRepo", "~/git/drat"),
```

Comment:
shouldn't this be `file.path()` rather than `/` ?

Reply:
Not sure what you mean. The first argument of `pruneRepo` was not changed.

Reply:
good then